ajax/libs/element-ui/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "element-ui",
+  "version": "1.0.0-rc.7",
+  "description": "A Component Library for Vue.js.",
+  "filename": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:ElemeFE/element.git"
+  },
+  "author": "ElemeFE",
+  "homepage": "http://element.eleme.io",
+  "keywords": [
+    "eleme",
+    "vue",
+    "components"
+  ],
+  "license": "MIT",
+  "npmName": "element-ui",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "index.js",
+        "theme-default/+(index.css|fonts/*)"
```

Comment:
OK, thank you. I add project first for cdnjs, so some rule I may mistake. If you find some error, please help me correct them.